File: src/jc_quotes.py
from requests import get

class JCQuotes:
	def __init__(self) -> None:
		self.api = "https://www.jcquotes.com/api"
		self.headers = {
			"user-agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/73.0.3683.86 Chrome/73.0.3683.86 Safari/537.36"
		}

	def get_random_quote(self) -> dict:
		return get(
			f"{self.api}/quotes/random",
			headers=self.headers).json()

File: src/jc_quotes.js
class JcQuotes {
	constructor() {
		this.api = "https://www.jcquotes.com/api"
		this.headers = {
			"user-agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
		}
	}

	async getRandomQuote() {
		const response = await fetch(
			`${this.api}/quotes/random`, {
				method: "GET",
				headers: this.headers
			})
		return response.json()
	}
 }

module.exports = {JcQuotes}
